core/src/main/java/org/eobjects/datacleaner/extension/output/CreateCsvFileAnalyzer.java

```diff
@@ -107,16 +127,94 @@ public OutputWriter createOutputWriter() {
         for (int i = 0; i < headers.length; i++) {
             headers[i] = columns[i].getName();
         }
-        return CsvOutputWriterFactory.getWriter(file.getPath(), headers, separatorChar, quoteChar, escapeChar,
-                includeHeader, columns);
+        return CsvOutputWriterFactory.getWriter(_tempFile.getPath(), headers, separatorChar, quoteChar, escapeChar, includeHeader, columns);
     }
 
     @Override
     protected WriteDataResult getResultInternal(int rowCount) {
+        final CsvConfiguration csvConfiguration = new CsvConfiguration(CsvConfiguration.DEFAULT_COLUMN_NAME_LINE, FileHelper.DEFAULT_ENCODING,
```

Comment:
Should also take into account whether or not we need to do sorting. If not, a lot of this can be left out.